vATIS.Desktop/Weather/MockMetarRepository.cs

```diff
@@ -0,0 +1,38 @@
+using System.Threading.Tasks;
+using ReactiveUI;
+using Vatsim.Vatis.Events;
+using Vatsim.Vatis.Io;
+using Vatsim.Vatis.Weather.Decoder.Entity;
+
+namespace Vatsim.Vatis.Weather;
+
+public class MockMetarRepository : IMetarRepository
+{
+    private readonly IDownloader mDownloader;
+    private readonly Decoder.MetarDecoder mMetarDecoder;
+    private const string LOCAL_METAR_SERVICE_URL = "http://localhost:5500/metar?id=";
```

Comment:
```suggestion
    private const string LOCAL_METAR_SERVICE_URL = $`http://{IPAddress.Loopback.ToString()}:5500/metar?id=`;
```

Think this needs to be changed to ensure it works on MacOS, or at least to be consistent with the WebSocket server in how the localhost address is specified.